src/jit/lowerxarch.cpp

```diff
@@ -2511,15 +2515,30 @@ bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, Ge
                     break;
                 }
 
+                case NI_SSE2_Insert:
                 case NI_SSE41_Insert:
                 case NI_SSE41_X64_Insert:
                 {
                     if (containingNode->gtSIMDBaseType == TYP_FLOAT)
```

Comment:
This case was incorrectly allowing `LoadAligned` and `Load`, when it should have only allowed `LoadScalar` (since it only reads 32-bits when the operand comes from memory).